debian/install

```diff
@@ -0,0 +1,10 @@
+debian/postinst.d/10-groups-creator.sh var/lib/dpkg/info/ottopia-keepalived-exporter/postinst.d
+debian/postinst.d/11-user-creator.sh var/lib/dpkg/info/ottopia-keepalived-exporter/postinst.d
+debian/postinst.d/12-add-to-group.sh var/lib/dpkg/info/ottopia-keepalived-exporter/postinst.d
+debian/postinst.d/13-set-capabilities.sh var/lib/dpkg/info/ottopia-keepalived-exporter/postinst.d
+debian/postinst.d/14-python-packages.sh var/lib/dpkg/info/ottopia-keepalived-exporter/postinst.d
+debian/postinst.d/utils/constants.sh var/lib/dpkg/info/ottopia-keepalived-exporter/postinst.d/utils
+debian/postinst.d/utils/logger.sh var/lib/dpkg/info/ottopia-keepalived-exporter/postinst.d/utils
+ottopia-keepalived-exporter.service lib/systemd/system
+venv/lib/python3.9/site-packages/ottopia_logging usr/lib/python3/dist-packages
```

Comment:
I would add the following file to the deb
ottopia_logging-0.1.1-py2.py3-none-any.whl
and install it using pip in the post install script.